action.yml

```diff
@@ -77,5 +79,9 @@ runs:
 
         # update the PR body to set versions if different than creation
         gh pr edit --body "${BODY}"
+        if [ -n "${{ inputs.label }}" ];
+        then
+          gh pr --label "${{ inputs.label }}"
```

Comment:
Does this return a non-zero exit code if the PR already has this label?

Also note the behavior here is such that if you remove the label, the next time this task runs it will add the label back.